elrond-wasm-debug/src/execute_mandos.rs

```diff
@@ -77,6 +77,22 @@ fn parse_execute_mandos_steps(
 					if let Some(u64_value) = &block_info_obj.block_round {
 						state.previous_block_info.block_round = u64_value.value;
 					}
+					if let Some(bytes_value) = &block_info_obj.block_random_seed {
+						const SEED_LEN: usize = 48;
+						let val = &bytes_value.value;
+
+						assert!(val.len() <= SEED_LEN, "block random seed input value too long!");
+
+						let mut seed = [0u8; SEED_LEN];
+						let mut temp = Vec::with_capacity(SEED_LEN);
```

Comment:
you don't need an extra vec.

Just do something like
&seed[SEED_LEN - val.len()..].copy_from_slice(bytes_value.as_slice());
state.previous_block_info.block_random_seed = Box::from(seed);